web/templates/scheduled_messages_indicator.hbs

```diff
@@ -0,0 +1,7 @@
+
+<div class="scheduled_message_indicator">
+    {{#tr}}
+    You have <z-link>{scheduled_message_count, plural, =1 {1 scheduled message} other { # scheduled messages}}</z-link> for this conversation.
```

Comment:
Is there supposed to be a space between `{ #`?

Reply:
Yeah, the linter picks it up as a unclosed comment and I'm not exactly sure how to escape it. By putting the space, it works just fine.

Reply:
Ahh, the lint rule is bugged (it is looking for Django template engine comments despite this being a Handlebars file).

Added a commit to fix this.